src/basilisp/cli.py

```diff
@@ -78,6 +79,19 @@ def bootstrap_repl(ctx: compiler.CompilerContext, which_ns: str) -> types.Module
     return importlib.import_module(REPL_NS)
 
 
+def init_path(args: argparse.Namespace) -> None:
+    def append_once(path: str) -> None:
```

Comment:
Should this be `prepend_once` to better reflect the operation, since order is important in the `sys.path` list?

Unfortunately, this suggestion impacts the rest of the patch. Perhaps another solution would be to add a flag to this function, allowing it to specify whether to append or prepend, to handle the `append_once("")` case?

Reply:
Yes you're right. I was initially appending and then realized that prepending was the better choice here. I will update shortly.

Based on your comments in the ticket, I'm unsure if I should retain the `-p` option still or if I should only prepend `""`. Let me know your preference.

Reply:
Sorry I wasn't clear. What I meant is that if we change `append_once` to `prepend_once`, the rest of the patch, including the new option, might also need to change its naming to prepend—though I could be wrong. I find the new option valuable, as it resolves the issue of not being able to set the `PYTHONPATH` environment variable inline with the basilisp command on MS-Windows, at least for adding local dirs to the `PYTHONPATH`. thanks